samples/nrf9160/nidd/src/main.c

```diff
@@ -46,10 +47,17 @@ static void lte_handler(const struct lte_lc_evt *const evt)
 
 static void modem_init(void)
 {
-	int err = lte_lc_init();
+	int err;
+
+	err = nrf_modem_lib_init(NORMAL_MODE);
+	if (err) {
+		printk("Modem library initialization failed, error: %d\n", err);
+		return;
+	}
 
+	err = lte_lc_init();
 	if (err) {
-		printk("Modem initialization failed, error: %d\n", err);
+		printk("Modem link initialization failed, error: %d\n", err);
```

Comment:
I guess `LTE connection` or something like that is more commonly used than `modem link`, but not a big deal